Add membership commands in comands topic (#4551)
Codecademy/docs: content/python/concepts/operators/operators.md

- `and`
- `or`

> ## Membership operators

Reviewer: ```suggestion
## Membership operators

```

- `and`
- `or`

## Membership operators

> - The `in` operator returns `True` if the element on the right _contains_ the element in the left. Here, the element on the right side of the `in` operator must be an iterable object like a list, string, dictionary, etc.

Reviewer: ```suggestion
- The `in` operator returns `True` if the element on the left is found within the iterable object on the right. Here, the element on the right side of the `in` operator must be an iterable object like a list, string, dictionary, etc.
```

> - The `not in` operator returns `False` if the element on the right _does not contain_ the element on the left. Again, the element on the right side of the ` not in` operator must be an iterable object.

Reviewer: ```suggestion
- The `not in` operator returns `False` if the element on the left is not found within the iterable object on the right. Again, the element on the right side of the `not in` operator must be an iterable object.
```
